src/state/UserData.js

```diff
@@ -3,13 +3,30 @@ const onReturn = Symbol('onReturn');
 const READ = 'userDataRead';
 const WRITE = 'userDataWrite';
 const DELETE = 'userDataRemove';
+
+const IDBOPEN = 'IDBOpen';
+const IDBADD = 'IDBAdd';
+const IDBREMOVE = 'IDBRemove';
+const IDBREAD = 'IDBRead';
+const IDBUPDATE = 'IDBUpdate';
+const IDBCLOSE = 'IDBClose';
+const IDBREADALL = 'IDBReadAll';
 /**
  *
  * Manages data between SpringRoll Container and SpringRoll
  * @export
  * @class UserData
  */
 export class UserData {
+
+  /**
+   * 
+   * @param {*} dbName 
+   */
+  constructor (dbName) {
```

Comment:
Is the constructor actually required anywhere? Since they're specifying dbName in the IDBOpen() method anyway, we could remove the constructor and keep all the methods static.